hedgehog.cabal

```diff
@@ -34,15 +34,17 @@ source-repository head
 library
   build-depends:
       base                            >= 3          && < 5
-    , base64-bytestring               >= 1.0        && < 1.1
+    , ansi-terminal                   >= 0.6        && < 0.7
```

Comment:
I know it would be less convenient, but I wonder whether it would be worth splitting up the project into core, runner-terminal, example, etc. I suspect that will be too annoying for the first release.